decidim-core/config/locales/en.yml

```diff
@@ -361,6 +361,7 @@ en:
       document:
         download: Download file
       documents:
+        component_documents: "%{name} documents"
```

Comment:
This generates a bug when it's translated, as for instance in Catalan would be "Documents de les %{name}" and that generates a mini-bug with the name of the document and the usage of the uppercase in the first letter:
![Screenshot of the bug](https://github.com/decidim/decidim/assets/717367/cb9c9406-a68d-42df-ba04-d73381250bb6)

I think it's better to have explicitly the component name so translators can choose their own usage of lower/upper cases. 

```yaml
      documents:
        component_documents:
          meetings: Meetings documents
 ```

You'll need to change the `name` method in `ComponentAttachmentCollectionPresenter` of course

Reply:
I have added the new title. To avoid ambiguities if there is more than one meetings component collection to show in the space component block the component name is added to the title separated with a "-". If there is only one to show only "Meetings documents" is displayed.

Reply:
>  To avoid ambiguities if there is more than one meetings component collection to show in the space component block the component name is added to the title separated with a "-". 

Ok, I think the cases where there are different meetings with components and those meetings have attachments are too few. I'll wait for a bug report if someone complains about this to have more feedback :)